### PR TITLE
Fix uploading compendia by using/setting the correct bucket env var.

### PR DESCRIPTION
**Pull request by kurtwheeler**
…

## Issue Number

#1258 

## Purpose/Implementation Notes

Also fix GEOParse cleanup.

### CODE REVIEW

File: foreman/data_refinery_foreman/foreman/management/commands/correct_affy_cdfs.py [hunk 1]
```diff
@@ -92,7 +92,13 @@ def handle(self, *args, **options):
                     # GEOparse downloads files here and never cleans them up! Grrrr!
                     download_path = GEO_TEMP_DIR + experiment.accession_code + '_family.soft.gz'
                     # It's not a directory, but ignore_errors is useful.
-                    shutil.rmtree(download_path, ignore_errors=True)
+                    try:
+                        os.remove(download_path, ignore_errors=True)
+                    except:
```
Comment (arielsvn) [inline foreman/data_refinery_foreman/foreman/management/commands/correct_affy_cdfs.py]: do we need the `try...except` if we have `ignore_errors=True`?

Also, `except...pass` makes me nervous.

Comment (kurtwheeler) [reply foreman/data_refinery_foreman/foreman/management/commands/correct_affy_cdfs.py]: Oh good catch! `ignore_errors=True` isn't an option on os.remove(), I just forgot to delete it from the other method. I've changed this so we log so we can tell why we're failing to delete stuff.

File: foreman/data_refinery_foreman/foreman/management/commands/correct_affy_cdfs.py [hunk 2]
```diff
@@ -92,7 +92,13 @@ def handle(self, *args, **options):
...
                     download_path = GEO_TEMP_DIR + experiment.accession_code + '_family.soft.gz'
                     # It's not a directory, but ignore_errors is useful.
-                    shutil.rmtree(download_path, ignore_errors=True)
+                    try:
+                        os.remove(download_path)
+                    except:
+                        # Don't anything interrupt this, like say,
+                        # GEOParse downloading a directory instead of
+                        # a file...
+                        logger.exception("Failed to delete an archive.")
```
Comment (arielsvn) [inline foreman/data_refinery_foreman/foreman/management/commands/correct_affy_cdfs.py]: Does this include the exception that was raised?

Comment (kurtwheeler) [reply foreman/data_refinery_foreman/foreman/management/commands/correct_affy_cdfs.py]: Yup, that's what `logger.exception` does.

**Pull request merged**